mne/export/_egimff.py

```diff
@@ -48,6 +53,15 @@ def export_evokeds_mff(fname, evoked, history=None, *, verbose=None):
     sampling_rate = int(info['sfreq'])
 
     # Initialize writer
+    # Future changes: conditions based on version or mffpy requirement if
+    # https://github.com/BEL-Public/mffpy/pull/92 is merged and released.
+    try:
+        fname = _check_fname(fname, overwrite=False)
+    except FileExistsError:
+        if overwrite:
+            shutil.rmtree(fname)
+        else:
+            raise
```

Comment:
It looks like this doesn't actually *use* the `overwrite` parameter. How about:
```suggestion
    fname = _check_fname(fname, overwrite=overwrite)
    if op.exists(fname):
        os.remove(fname) if op.isfile(fname) else shutil.rmtree(fname)
```

Reply:
It used the argument in the except, but that works too and is cleaner!